nmmo/core/action.py

```diff
@@ -243,26 +243,26 @@ def call(realm, entity, style, target):
       target.history.time_alive < immunity:
       return None
 
-    #Check if self targeted
-    if entity.ent_id == target.ent_id:
+    #Check if self targeted or target already dead
+    if entity.ent_id == target.ent_id or not target.alive:
       return None
 
-    #Can't attack out of range
+    #Can"t attack out of range
     if utils.linf_single(entity.pos, target.pos) > style.attack_range(config):
       return None
 
     #Execute attack
     entity.history.attack = {}
-    entity.history.attack['target'] = target.ent_id
-    entity.history.attack['style'] = style.__name__
+    entity.history.attack["target"] = target.ent_id
+    entity.history.attack["style"] = style.__name__
     target.attacker = entity
     target.attacker_id.update(entity.ent_id)
 
     from nmmo.systems import combat
     dmg = combat.attack(realm, entity, target, style.skill)
 
-    if style.freeze and dmg > 0:
-      target.status.freeze.update(config.COMBAT_FREEZE_TIME)
+    # if style.freeze and dmg > 0:
```

Comment:
did you mean to leave this commented? if so, explain why

Reply:
Removed these completely, as there is no freeze attack.